### PR TITLE
Using function instead of `arrayMove` from `react-sortable-hoc`.

### PR DESCRIPTION
**Pull request by dennisoelkers**
## Motivation and Context
Before this change, `arrayMove` was used from the `react-sortable-hoc`
package. It will not be exported from it in the future anymore, so this
change is now adding a simple function doing the same.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: graylog2-web-interface/src/views/components/aggregationbuilder/SortableSelect.jsx [hunk 1]
```diff
@@ -2,14 +2,20 @@ import React from 'react';
...
 import { findIndex } from 'lodash';
 
 import Select from 'views/components/Select';
 import styles from './SortableSelect.css';
 
 const SortableValueList = SortableContainer(components.ValueContainer);
 
+const arrayMove = (array, from, to) => {
+  const result = array.slice();
+  result.splice(to < 0 ? array.length + to : to, 0, array.splice(from, 1)[0]);
```
Comment (kmerz) [inline graylog2-web-interface/src/views/components/aggregationbuilder/SortableSelect.jsx]: result.splice(to < 0 ? array.length + to : to, 0, result.splice(from, 1)[0]);

File: graylog2-web-interface/src/views/components/aggregationbuilder/SortableSelect.jsx [hunk 2]
```diff
@@ -2,14 +2,20 @@ import React from 'react';
...
+  return array;
```
Comment (kmerz) [inline graylog2-web-interface/src/views/components/aggregationbuilder/SortableSelect.jsx]: return result;

File: graylog2-web-interface/src/views/components/aggregationbuilder/SortableSelect.jsx (hunk 1 above)
Comment (kmerz) [inline graylog2-web-interface/src/views/components/aggregationbuilder/SortableSelect.jsx]: result.splice(to < 0 ? array.length + to : to, 0, result.splice(from, 1)[0]);

Comment (dennisoelkers) [reply graylog2-web-interface/src/views/components/aggregationbuilder/SortableSelect.jsx]: Seems like at least one test case would be nice! :D

**Pull request merged**